shell/common/engine.cc

```diff
@@ -450,6 +448,13 @@ void Engine::StartAnimatorIfPossible() {
     animator_->Start();
 }
 
+std::string Engine::DefaultRouteName() {
+  if (!initial_route_.empty()) {
+    return initial_route_;
+  }
+  return std::string("/");
```

Comment:
No need for the explicit call to `std::string`.  The compiler will add it for you.